hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/validators/TokenUpdateValidator.java

```diff
@@ -108,8 +108,10 @@ private ExpiryMeta resolveExpiry(
             @NonNull final Token token,
             @NonNull final TokenUpdateTransactionBody op,
             @NonNull final ExpiryValidator expiryValidator) {
-        final var givenExpiryMeta =
-                new ExpiryMeta(token.expiry(), token.autoRenewSecs(), token.autoRenewAccountNumber());
+        final var givenExpiryMeta = new ExpiryMeta(
+                token.expiry(),
+                token.autoRenewSecs(),
+                token.autoRenewAccountId().accountNum());
```

Comment:
Once ExpiryMeta is changed:

```suggestion
                token.autoRenewAccountId());
```

Reply:
Updated, thanks.